### ISSUE TITLE
[Feature]: scikit-learn preprocessing module support

### ISSUE THREAD

**Issue by AHReccese** [openscilab/pymilo#88]
### Describe the feature you want to add

Support serialization and deserialization of all Scikit-learn [preprocessing transformers](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.preprocessing)
* Binarizer 
* FunctionTransformer
* KBinsDiscretizer
* KernelCenterer
* LabelBinarizer 
* LabelEncoder
* MultiLabelBinarizer
* MaxAbsScaler
* Normalizer
* OneHotEncoder
* OrdinalEncoder
* PolynomialFeatures
* PowerTransformer
* QuantileTransformer
* RobustScaler
* SplineTransformer
* StandardScaler
* TargetEncoder
